app/Services/Campaign/ExportService.php

```diff
@@ -316,33 +364,44 @@ protected function processImage(Image $image): self
 
     protected function process(string $entity, $model): self
     {
+
+        if ($model instanceof Entity) {
+            $modelEntity = $model;
```

Comment:
$modelEntity is hard to read when in the rest of Kanka we have $entity. Refactor it instead to use $entity, and change the function's $entity variable to $module instead to make it cleaner